.github/workflows/workflow.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           scope: '@nciocpl'
       ## Override default lerna version
       - name: Install Lerna
-        run: yarn global add lerna@6.6.2
+        run: yarn global add lerna@5.0.0
```

Comment:
update generate-bs-images.yml too

Reply:
should probably match devDependencies in package.json since we're at it...

```
	"devDependencies": {
		"lerna": "6.6.2",
```

Reply:
Updated the bs workflow. Negative to matching dependencies - we want 6.6.2, or newer. We just need this to be able to bootstrap, which downloads the newer version that ultimately is used.